src/view/global.styl

```diff
@@ -23,6 +23,16 @@ code {
   font-family: monospace;
 }
 
+.u-toolTipMinimumSize {
+  min-width:  120*$px !important;
+}
+
+// This provides enough padding to the right of the 'learn more' link on events
+// to allow the error toolTip to dispaly on the right side of the text field
```

Comment:
typo: display

Reply:
removed this comment as the style was implemented differently